src/linker/Linker/LinkerFatalErrorException.cs

```diff
@@ -31,9 +20,6 @@ public LinkerFatalErrorException (MessageContainer message)
 			MessageContainer = message;
```

Comment:
Nit: We should also check that `Code != null`

Reply:
This should be covered by the category check since the only way to create an error `MessageContainer` is with a ctor that requires a code - but an extra check doesn't hurt and might prevent future bugs.

Reply:
I was asking purely because we assert on it when consuming the exception...